stackslib/src/chainstate/coordinator/tests.rs

```diff
@@ -520,6 +520,17 @@ impl RewardSetProvider for StubbedRewardSetProvider {
             signers: None,
         })
     }
+
+    fn get_reward_set_nakamoto(
+        &self,
+        cycle_start_burn_height: u64,
+        chainstate: &mut StacksChainState,
+        burnchain: &Burnchain,
+        sortdb: &SortitionDB,
+        block_id: &StacksBlockId,
+    ) -> Result<RewardSet, CoordError> {
+        panic!("Stubbed reward set provider cannot be invoked in nakamoto")
```

Comment:
Is there a corresponding ticket for this?

Reply:
No, and there shouldn't be -- this is a struct only ever used in testing. If it would be helpful to test something in Nakamoto with a stubbed reward set provider, we can update this then.